kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/FacilityPermissionsForm.vue

```diff
@@ -90,11 +83,7 @@
     },
     methods: {
       focusOnTextbox() {
-        if (this.nonformalIsSelected) {
-          return this.$refs['facility-name-nonformal'].focus();
-        } else if (this.formalIsSelected) {
-          return this.$refs['facility-name-formal'].focus();
-        }
+        return this.$refs['facility-name'].focus();
```

Comment:
Could you remove the `watch.selected` handler on line 74? Otherwise, everytime you change between formal/non-formal, the Facility Name field gets focused

![image](https://user-images.githubusercontent.com/10248067/95110278-9f560880-06f2-11eb-93b8-1854b0eb48b2.png)

Reply:
Done